x-pack/legacy/plugins/canvas/public/components/sidebar_header/sidebar_header.tsx

```diff
@@ -293,7 +293,7 @@ export class SidebarHeader extends Component<Props, State> {
             onClick: bringToFront,
           },
           {
-            name: shortcutHelp.BRING_TO_FRONT, // TODO: same as above
+            name: shortcutHelp.BRING_FORWARD, // TODO: same as above
```

Comment:
nit: is the `TODO` comment still necessary...?

Reply:
Yeah, it's referring to the comment on line 291 where I left a note that we should disable these controls if the element is the top most element.